tests/tests_selinux_modules_checksum.yml

```diff
@@ -0,0 +1,56 @@
+---
+- name: Test management of SELinux modules
+  hosts: all
+  become: true
+  become_method: sudo
+  become_user: root
+  gather_facts: true
+  tasks:
+    - name: Execute the role and catch errors
+      when: ansible_distribution == "Fedora" or
+        (ansible_distribution_major_version | int > 8 and
+        ansible_distribution in ["CentOS", "RedHat", "Rocky"]) or
+        (ansible_distribution_major_version | int == 8 and
```

Comment:
did you mean `int <= 8` here?

For this `when` condition, do you want to execute on Fedora, or on Red Hat family OS with version in [6, 7, 8, 9]?